client.go

```diff
@@ -67,6 +67,26 @@ type User struct {
 	Badges      map[string]int
 }
 
+func (u User) IsBroadcaster() bool {
+	return u.hasRole("broadcaster")
+}
+
+func (u User) IsModerator() bool {
+	return u.hasRole("moderator") || u.IsBroadcaster()
```

Comment:
This makes no sense. If you would want this logic in your client consuming code and not in the library. A Broadcaster is not a moderator, even though he has moderation permissions.